### DISCUSSION TITLE
How to config files match filename, no path, like intellj idea find file?

### DISCUSSION THREAD

**Discussion by hyawara** [ibhagwan/fzf-lua#1434] [Q&A] [answered]
intellj idea find:
![image](https://github.com/user-attachments/assets/dcbcc2bf-5d97-4380-8520-f7bced74a46b)

fzf-lua find:
![image](https://github.com/user-attachments/assets/02fe45a6-af81-4de2-9529-e5bb290073b1)

**Comment by ibhagwan** [comment, marked as answer]
Use fzf's `--nth` combined with a tab delimiter to limit the search to filename only:
```lua
:lua require("fzf-lua").files({formatter="path.filename_first",fzf_opts={["--delimiter"]="[\t]",["--nth"]=1}})
```

**Reply by hyawara** [reply]
Oh my god, thank you very much!

**Reply by hyawara** [reply]
Then how can I customize the highlights here? thank you

![image](https://github.com/user-attachments/assets/5a1f5a69-27a2-4ea2-ba31-973a0f8b7725)

**Reply by ibhagwan** [reply]
https://github.com/ibhagwan/fzf-lua?tab=readme-ov-file#fzf-colors
